doc/ejabberd-2.1.8-OLD/guide.tex

```diff
@@ -81,6 +81,7 @@
 \newcommand{\modoffline}{\module{mod\_offline}}
 \newcommand{\modofflineodbc}{\module{mod\_offline\_odbc}}
 \newcommand{\modping}{\module{mod\_ping}}
+\newcommand{\modpubsub}{\module{mod\_pubsub}}
```

Comment:
I think don't have to care about the old doc in tex. It's probably good time to remove it completely, but that's a subject for a separate PR.